tests/core/conversion/converters/test_reflection_pad.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <string>
+#include "core/compiler.h"
+#include "gtest/gtest.h"
+#include "tests/util/util.h"
+#include "torch/csrc/jit/ir/irparser.h"
+
+TEST(Converters, ATenReflection_pad2dTensorConvertsCorrectly) {
+  const auto graph = R"IR(
+      graph(%0 : Tensor):
+        %1 : int = prim::Constant[value=1]()
+        %2 : int = prim::Constant[value=1]()
+        %3 : int = prim::Constant[value=2]()
+        %4 : int = prim::Constant[value=0]()
+        %5 : int[] = prim::ListConstruct(%1, %2, %3, %4)
+        %6 : Tensor = aten::reflection_pad2d(%0, %5)
+        return (%6))IR";
+
+  auto g = std::make_shared<torch::jit::Graph>();
+  torch::jit::parseIR(graph, g.get());
+
+  auto in1 = at::randint(1, 10, {1, 3, 5, 5}, {at::kCUDA});
+
+  auto params = torch_tensorrt::core::conversion::get_named_params(g->inputs(), {});
```

Comment:
`get_named_params` is now replaced with `get_static_params`. Please change this and other similar lines with 
` params = torch_tensorrt::core::ir::get_static_params(g->inputs(), {});`